src/pages/user/UserInstitution.tsx

```diff
@@ -87,9 +103,11 @@ const UserInstitution: FC = () => {
   return (
     <Card>
       <Heading>{t('heading.organizations')}</Heading>
-      {units?.map((unit: Unit, index: number) => (
-        <InstitutionCard key={index} unit={unit} />
-      ))}
+      {units.length > 0 ? (
+        units.map((unit: Unit, index: number) => <InstitutionCard key={index} unit={unit} />)
```

Comment:
sikket kommentert før, men burde vi ikke brukt et mer spesifikt begrep for Unit  (som f.ex institution-unit) ?

Reply:
jo, det er et godt poeng. Kan godt rename typene som har Unit i seg til å være InstitutionUnit 👍

Reply:
Vakkert :-)